src/worker.rs

```diff
@@ -88,22 +89,23 @@ impl Worker {
 
             let policy_execution_mode = policy_metadata.unwrap_or_default().execution_mode;
 
-            let mut policy_evaluator = match PolicyEvaluatorBuilder::new(id.to_string())
-                .policy_contents(&policy_contents)
-                .execution_mode(policy_execution_mode)
-                .settings(settings_json)
-                .callback_channel(callback_handler_tx.clone())
-                .build()
-            {
-                Ok(policy_evaluator) => policy_evaluator,
-                Err(err) => {
-                    evs_errors.insert(
-                        policy.url.clone(),
-                        format!("could not instantiate policy: {:?}", err),
-                    );
-                    continue;
-                }
-            };
+            let mut policy_evaluator =
+                match PolicyEvaluatorBuilder::new(id.to_string(), policy.mutating)
```

Comment:
I'm not very happy with the necessity to send the flag for the `PolicyEvaluatorBuilder`. But I could not find a better solution for that. Considering the time that I have before vacations (today is 12/31/2021) I'm not sure if I find be able to find a better solution in time. So, I'm opening the current code to feedback from other colleagues.

Reply:
I think this is fine, I would however make that a dedicated method of the policy (`downgrade_to_validation()` with no params).

As written above, I think we should by default do what the policy author wanted, and then in special cases force this downgrade. Basically, the settings entries should by default fall back to do what the policy author intended.

You will see that as part of the GH review I'm authoring

Reply:
Maybe a better name could be `disable_mutations`...

Reply:
I think we can avoid adding this field to the PolicyEvaluatorBuilder. I think it has to be abstracted from the evaluator itself, given it's not its responsibility. This is my proposal on top of this PR: https://gist.github.com/ereslibre/819d94af9445a28ca761fb1d1111871c

Reply:
Also, I agree that the setting can be optional and default to the policy behavior if it's omitted.

Reply:
I've updated the branch with the changes suggested by @ereslibre